app/views/foreman/job_templates/remove_packages_by_search_query_-_katello_ansible_default.erb

```diff
@@ -0,0 +1,26 @@
+<%#
+kind: job_template
+name: Remove packages by search query - Katello Ansible Default
+job_category: Katello
+description_format: 'Remove package(s) %{Packages search query}'
+feature: katello_package_remove_by_search
+provider_type: Ansible
+template_inputs:
+- name: Packages search query
+  description: Filter criteria for packages to be removed.
+  input_type: user
+  required: false
+%>
+<% package_names = @host.package_names_for_job_template(
+  action: 'remove',
+  search: input('Packages search query')
+) -%>
+---
+- hosts: all
+  tasks:
+    - package:
+        name: 
+<% package_names.each do |package_name| -%>
+          - <%= package_name %>
+<% end -%>
+        state: absent
```

Comment:
```suggestion
<%= render_template('Package Action - Ansible Default', :state => 'absent', :name => package_names) %>
```

Reply:
I didn't do this because I don't think that `:name => package_names` will render a correct YAML list.

Reply:
IIRC it generate a valid YAML list (the compact form) each package name separated by comma. Ansible understand it. I will recheck.

Reply:
Rendered output:
```yaml

# For Windows targets use the win_package module instead.
---
- hosts: all
  tasks:
    - package:
        name: ["libssh2", "libssh-devel"]
        state: absent
```

query: name = libssh-devel or name = libssh2
We have a trailling line empty but that can be changed by adding minus

I launched the jobs and confirm that Ansible is able to read this YAML.